app/src/components/common/icons/IconClose.tsx

```diff
@@ -23,10 +23,11 @@ interface Props {
 }
 
 export const IconClose = (props: Props) => {
-  const { color = '#37474F', hoverEffect = false, onClick, size = '24' } = props
+  const { color = '#37474F', hoverEffect = true, onClick, size = '24' } = props
 
   return (
     <Wrapper
+      color={'inherit'}
```

Comment:
No need for color here....

Reply:
Here I added inherit so that the SVG would get the color from [that](https://github.com/protofire/omen-exchange/pull/2062#discussion_r661726708) wrapper. I tested it and it worked well that way. Now I will search for a better way from our codebase